message_parser/pom.xml

```diff
@@ -4,7 +4,7 @@
   <groupId>gov.nasa.earthdata</groupId>
   <artifactId>cumulus-message-adapter</artifactId>
   <packaging>jar</packaging>
-  <version>1.2.4</version>
+  <version>1.2.4-SNAPSHOT</version>
```

Comment:
Using snapshot prevents the error: `Failed to deploy metadata: Could not transfer metadata gov.nasa.earthdata:cumulus-message-adapter/maven-metadata.xml from/to clojars (https://clojars.org/repo/): Access denied to: https://clojars.org/repo/gov/nasa/earthdata/cumulus-message-adapter/maven-metadata.xml, ReasonPhrase: Forbidden - redeploying non-snapshots is not allowed (see https://git.io/v1IAs)`